frontend/sw360-portlet/src/main/resources/META-INF/resources/html/components/includes/releases/summaryRelease.jspf

```diff
@@ -76,14 +80,16 @@
 
         <tr>
             <td><liferay-ui:message key="main.licenses" />:</td>
-            <td>
-                <sw360:DisplayCollection value="${release.mainLicenseIds}"/>
+            <td class="actions">
+<%--                 <sw360:DisplayCollection value="${release.mainLicenseIds}"/> --%>
```

Comment:
please remove the commented code, same for line 91.

Reply:
removed the commented line.

Reply:
Code looks good, Please merge it.